include/quic/endpoint.hpp

```diff
@@ -171,6 +171,8 @@ namespace oxen::quic
 
         void close_connection(Connection& conn, int code = NGTCP2_NO_ERROR, std::string_view msg = "NO_ERROR"sv);
 
+        void close_connection(ConnectionID cid, int code = NGTCP2_NO_ERROR, std::string_view msg = "NO_ERROR"sv);
```

Comment:
It looks like `Connection::close_connection` is calling the previous declaration of this, so this appears unused?

Reply:
It’s not unused, closing connections via SCID is useful for lokinet integration

Reply:
Ah okay, it's just unused in this commit; that makes more sense.

Reply:
Oh i understand what you meant now! Yes exactly it’s for external use. Will PM you